backend/src/org/akvo/lumen/endpoint/dashboard.clj

```diff
@@ -1,183 +1,33 @@
 (ns org.akvo.lumen.endpoint.dashboard
   (:require [compojure.core :refer :all]
-            [clojure.java.jdbc :as jdbc]
-            [hugsql.core :as hugsql]
             [org.akvo.lumen.component.tenant-manager :refer [connection]]
-            [org.akvo.lumen.util :refer [squuid]]
-            [ring.util.response :as resp]))
+            [org.akvo.lumen.lib.dashboard :as dashboard]
```

Comment:
@kardan You chose `lib` instead of `domain`, but we still tend to talk about "domain" in our chats/calls.. Is there a specific reason you prefer this terminology?